src/CaptureFile/CaptureFile.cpp

```diff
@@ -81,6 +88,9 @@ class CaptureFileImpl : public CaptureFile {
   // message to detect the last message for the capture section.
   uint64_t capture_section_size_ = 0;
 
+  // This is the list of sections, which does not contain the section_list itself, nor the capture
+  // section. The section_list is ordered by section offset. Meaning a section with lower offset
+  // will come before a section with higher offset.
```

Comment:
I'm a bit confusing with this comment. So which sections this section list contains? It used to corresponding to the additional section list and so it was "the list of additional sections". Now it becomes "the list of additional sections and the user data section"?